### ISSUE TITLE
Translation: Italian

### ISSUE THREAD

**Issue by TechAle** [jwasham/coding-interview-university#1030]
Im willing to work to a translation in italian during my free time

**Comment by drew458** [comment]
Hi! i wanted to help for an italian translation as well. Lmk if u need

**Comment by TechAle** [comment]
@drew458 I'll start working on it when my exams are done, lets work on my fork https://github.com/TechAle/coding-interview-university
I'm sending the invite for editing the fork directly

**Comment by volpoh** [comment]
Hi guys, I started an italian translation too but I didn't see your work. Is it too late to join you?

**Comment by jwasham** [comment]
Yup I don't see any translation started. Anyone can start:
https://github.com/jwasham/coding-interview-university/issues/125

**Comment by volpoh** [comment]
@jwasham I should've uploaded my work-in-progress translation. I made it to line 842.
[You can find the translation here](https://github.com/volpoh/coding-interview-university/blob/main/translations/README-it.md)

**Comment by jwasham** [comment]
In progress is OK too. Just open a pull request.

**Comment by volpoh** [comment]
@jwasham OK, opened it. [Here](https://github.com/jwasham/coding-interview-university/pull/1146)

**Comment by cilippofilia** [comment]
I'd love to chip in and help! are the links still active?

**Comment by volpoh** [comment]
@cilippofilia Yep! :P I'm still working on it, but much has been done. [You can find my work here](https://github.com/volpoh/coding-interview-university/blob/main/translations/README-it.md) !

**Comment by marcomarrelli** [comment]
Good evening everyone! 😺 I've took the italian translation in hand again after almost two years (due to important personal commitments).
I've translated everything beside the optional part (line 1314).
If anyone still would like to get involved in the translation, you can find my work (here)[https://github.com/marcomarrelli/coding-interview-university].

**Comment by jwasham** [comment]
Welcome back!
